bark/world/evaluation/rss/rss_interface.hpp

```diff
@@ -0,0 +1,249 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+// Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#ifndef BARK_WORLD_EVALUATION_RSS_INTERFACE_HPP_
+#define BARK_WORLD_EVALUATION_RSS_INTERFACE_HPP_
+
+#include <assert.h>
+#include <fstream>
+#include <optional>
+#include <streambuf>
+#include <string>
+
+#include "bark/geometry/line.hpp"
+#include "bark/geometry/polygon.hpp"
+#include "bark/models/dynamic/dynamic_model.hpp"
+#include "bark/world/map/map_interface.hpp"
+#include "bark/world/objects/agent.hpp"
+#include "bark/world/world.hpp"
+
+#include <spdlog/spdlog.h>
```

Comment:
do they nor provide an option to include <as/rss/rss.hpp> that includes all files?

Reply:
Unfortunately they don't have it